packages/workbox-background-sync/test/browser/background-sync-queue.js

```diff
@@ -15,6 +15,10 @@
 /* global chai, workbox */
 
 'use strict';
+const testServerGen = require('../../../../utils/test-server-generator.js');
```

Comment:
This test makes no sense I'm afraid.

test files under **test/browser/ are unit tests that run in the browser. There is no require, there is no need to start a server and this code just fails :(